improve: use cache for nvm (#1924)
platformsh/platformsh-docs: docs/src/languages/nodejs/nvm.md

---
title: "Manage Node.js versions"
weight: 1
toc: false
description: See how to manage different Node.js versions in your Platform.sh containers.
---

> You may have instances where you need to use a specific version of Node.js that isn't available as a Platform.sh container.

Reviewer: ```suggestion
You may have instances where you need to use a specific version of Node.js that isn't available in an application container.
```
